test/unit/safeLogger.ts

```diff
@@ -12,24 +12,24 @@ describe('safeLogger', () => {
 
     it('should execute logger function', () => {
       let called = false;
-      const logger = {
+      const logger: any = {
```

Comment:
type Logger?

Reply:
The `Logger` type is not satisfied here (since we only implement `info`).

Reply:
how does that fit on `safeLogger(logger?: Logger)`? Logger | undefined != any

Reply:
I would like this solution more:

```ts
const logger = {
  info: () => {
    called = true;
  }
} as any as Logger;
```

It's a situation for type assertions not expanding a symbols type to anything.

Reply:
👍